doc/user/sql/create-schema.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "CREATE SCHEMA"
+description: "`CREATE SCHEMA` creates a new schema namespace."
```

Comment:
Similarly here, I think these should just be called "schemas", not "schema namespaces", as a "schema namespace" would be a schema nested within a schema (which is, of course, not supported).